src/components/history-list-page/HistoryPage.jsx

```diff
@@ -156,6 +158,34 @@ class HistoryPage extends React.Component {
             text: i18n.t("Metadata Types"),
             getValue: notification => this.getMetadataTypes(notification),
         },
+        {
+            name: "syncRule",
+            text: i18n.t("Sync Rule"),
+            getValue: ({ syncRule }) => this.getSyncRuleEditLink(syncRule),
```

Comment:
When checking the details of a non-rule sync, we see the placeholder "-" value for the "Sync Rule" attribute, which is a bit ugly. I see that I provided no way of hidden the field:

https://github.com/EyeSeeTea/d2-ui-components/blob/master/src/details-box/DetailsBox.component.js#L30

Not necessarily for now, but on the future it would be cool to simply don't render the field if `getValue` returns `undefined`. For now, we can return here `undefined` here and some time make the change in `DetailsBox`.

Reply:
What about:

- getValue: render getValue()
- primitive: print primite value
- null: Hide property
- undefined: print "-"

This way you can still print "-" for descriptions, and when you actually want to hide it you set it to null.

Reply:
LGTM!